kairos-prover/default.nix

```diff
@@ -77,7 +75,7 @@
     {
       devShells.risczero = pkgs.mkShell {
         RISC0_RUST_SRC = "${rustToolchain}/lib/rustlib/src/rust";
-        RISC0_DEV_MODE = 1;
+        RISC0_DEV_MODE = 0;
```

Comment:
Are we going to develop in non dev mode?

Reply:
We should always make sure that proving works.

Reply:
I think Avi encountered an error when using =1

Reply:
The contract cannot pickup devmode, so I set this to allow proofs to be submitted to the contract.

Reply:
proof gen is now around 6 seconds with metal